ntbs-service/Services/AuthorizationService.cs

```diff
@@ -119,8 +119,16 @@ public async Task<IQueryable<Notification>> FilterNotificationsByUserAsync(Claim
         public async Task<IList<Alert>> FilterAlertsForUserAsync(ClaimsPrincipal user, IList<Alert> alerts)
         {
             var userTbServiceCodes = (await _userService.GetTbServicesAsync(user)).Select(s => s.Code);
+            var userType = _userService.GetUserType(user);
             for (int i = alerts.Count - 1; i >= 0; i--)
             {
+                // For transfer alerts PHE users belonging to a PHEC cannot see and action transfer alerts as they are
+                // aimed to be actioned on a TB service level
+                if (userType == UserType.PheUser && (alerts[i].AlertType == AlertType.TransferRequest ||
+                                                     alerts[i].AlertType == AlertType.TransferRejected))
+                {
+                    alerts.RemoveAt(i);
+                }
                 if (alerts[i].TbServiceCode != null && !userTbServiceCodes.Contains(alerts[i].TbServiceCode))

```

Comment:
Maybe an `else if` to prevent the check being made if the alert has already been removed (and hypothetically throwing an exception).